examples/use-cases/push_events_to_syslog/README.md

```diff
@@ -0,0 +1,41 @@
+# Push Events to Syslog Daemon
+The push events to syslog daemon example provides the base for users to view a history of [events](https://developer.automox.com/openapi/axconsole/operation/getEvents/) happening on their Automox instance in their preferred syslog environments.
+
+There are two components that need to be ran:
+1. Automox Events Client ([`pull_events.py`](./pull_events.py))
+2. Syslog Server ([`syslog.py`](./syslog.py))
```

Comment:
This is only needed if testing it correct? Should we call that out and that if you have a Syslog endpoint for your SIEM already you point to that instead and this doesn't need run?

Reply:
I will add a note about testing here.